src/art/skypilot/backend.py

```diff
@@ -55,7 +55,7 @@ async def initialize_cluster(
         if resources is None:
             resources = sky.Resources(
                 cloud=sky.clouds.RunPod(),
-                accelerators={"H100": 1},
```

Comment:
Probably a bad idea since a lot of providers won't have that listed as an explicit type